### PR TITLE
[REVIEW] Fix polygon and polyline docstrings

### PR DESCRIPTION
**Pull request by trxcllnt**
Follow-up PR to https://github.com/rapidsai/cuspatial/pull/145

* Renames the `polyline_bounding_boxes` function's `R` parameter to `expansion_radius`
* Updates to the `polygon_bounding_boxes` and `polyline_bounding_boxes` C++ and Python docstrings

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

File: cpp/include/cuspatial/polyline_bounding_box.hpp [hunk 1]
```diff
@@ -28,20 +28,20 @@ namespace cuspatial {
...
  *
  * @return a cudf table of bounding boxes as four columns of the same type as `x` and `y`:
- * x1 - the lower-left x-coordinate of each bounding box
- * y1 - the lower-left y-coordinate of each bounding box
- * x2 - the upper-right x-coordinate of each bounding box
- * y2 - the upper-right y-coordinate of each bounding box
+ * x1 - the minimum x-coordinate of each bounding box
+ * y1 - the minimum y-coordinate of each bounding box
+ * x2 - the maximum x-coordinate of each bounding box
+ * y2 - the maximum y-coordinate of each bounding box
```
Comment (harrism) [inline cpp/include/cuspatial/polyline_bounding_box.hpp:37]:
```suggestion
 * x_min - the minimum x-coordinate of each bounding box
 * y_min - the minimum y-coordinate of each bounding box
 * x_max - the maximum x-coordinate of each bounding box
 * y_max - the maximum y-coordinate of each bounding box
```

File: python/cuspatial/cuspatial/core/gis.py [hunk 2]
```diff
@@ -259,18 +259,30 @@ def point_in_polygon(
...
+        Polygon point y-coordinates
 
-    returns
-    DataFrame of x1, y1, x2, y2 as minimum bounding boxes for each polygon
+    Returns
+    -------
+    result : cudf.DataFrame
+        minimum bounding boxes for each polygon
+
+        x1 : cudf.Series
```
Comment (harrism) [inline python/cuspatial/cuspatial/core/gis.py:278]: x_min, y_min etc?

File: python/cuspatial/cuspatial/core/gis.py [hunk 3]
```diff
@@ -280,24 +292,36 @@ def polygon_bounding_boxes(poly_offsets, ring_offsets, xs, ys):
...
+        radius of each polyline point
 
-    returns
-    DataFrame of x1, y1, x2, y2 as minimum bounding boxes for each polyline
+    Returns
+    -------
+    result : cudf.DataFrame
+        minimum bounding boxes for each polyline
+
+        x1 : cudf.Series
```
Comment (harrism) [inline python/cuspatial/cuspatial/core/gis.py:314]: x_min, y_min etc?

**Pull request merged**